src/main/java/seedu/address/model/person/Meetings.java

```diff
@@ -0,0 +1,93 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+
+import java.time.LocalDateTime;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import seedu.address.model.person.exceptions.TimeClashException;
+
+/**
+ * A list of meetings that ensures no overlaps in timings of all the meetings and does not allow nulls.
+ * A meeting clashes if the start time and end time overlaps with any meetings in the list.
+ * A meeting clash is determined using the {@code Meeting#isOverlap(Meeting)}. Thus, before every meeting is added to
+ * the list, Meeting#isOverlap(Meeting) is used to check for overlap.
+ *
+ * @see Meeting#isOverlap(Meeting)
+ */
+public class Meetings {
+
+    private final ObservableList<Meeting> internalList = FXCollections.observableArrayList();
```

Comment:
As the UI guy, thank you for implementing this with ObservableList from the start :)

Reply:
Npnp, one concern I do have tho, is that should I also, similar to the existing PersonList, make an immutable version of the list? Currently, the ObservableList is mutable